src/com/esotericsoftware/reflectasm/FieldAccess.java

```diff
@@ -572,5 +574,25 @@ static private MethodVisitor insertThrowExceptionForFieldType (MethodVisitor mv,
 		mv.visitInsn(ATHROW);
 		return mv;
 	}
-
+	
+	static public  FieldAccess getAccessUnsafe(Class<?> type)

```

Comment:
All right.And thanks for your notice! I'll try to improve it. Would you like me to write an inner-class in FieldAccess?